docs/notification-routing/configuring-sinks.rst

```diff
@@ -205,649 +205,6 @@ routed to the sink only from :ref:`customPlaybooks that explicitly name this sin
 
 Here too, matchers apply as usual and perform further filtering.
 
-Time-limiting sink activity
```

Comment:
Could you name it "Route Alerts by Time" so it matches the name of the other "Route Alerts by XXXX" pages?

Reply:
Feel free to shorten all the titles while you are at it to "Route by Time" etc